styles/style.css

```diff
@@ -1,15 +1,454 @@
-/* reset */
-
-html, body, div, span, iframe,
-h1, h2, h3, h4, h5, h6, p, a, img, q, strong, i, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, 
-figure, figcaption, footer, header, nav, section, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+
+.c-red {
+  color: #f72210;
+}
+
+.bg-red {
+  background-color: #f72210;
+  color: white;
+}
+
+main.container-fluid {
+  max-width: 1330px;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+/* nav */
+
+#nav-lg {
+  height: 108px;
+  padding: 20px 10px 15px;
+}
+
+.bottom-left div {
+  background-color: rgb(241, 241, 241);
+}
+
+.v-grid div:not(:last-child) {
+  border-bottom: 1px solid rgb(211, 210, 210);
+}
+
+.newsletter > div > div:last-child {
+  border: 1px solid rgb(209, 209, 209);
+}
+
+#nav-lg > div {
+  padding: 0 20px;
+}
+
+#main-logo {
+  width: 400px;
+  fill: white;
+  height: 60px;
+}
+
+#nav-lg-left div {
+  line-height: 2;
+  font-size: 15px;
+}
+
+footer img {
+  width: 240px;
+}
+
+.arrow > img {
+  padding-left: 10px;
+  height: 13px;
+}
+
+.nav-md-left img {
+  width: 170px;
+}
+
+.nav-sm-left img {
+  width: 165px;
+}
+
+.showcase-mini-story img {
+  float: left;
+  margin-right: 15px;
+}
+
+.editor-picks-article img {
+  width: 100%;
+}
+
+.show-case-article a img {
+  width: 100%;
+}
+
+.v-grid div img {
+  float: left;
+}
+
+.different-articles-container > div > img {
+  width: 100%;
+}
+
+.bottom-left-img img:first-child {
+  z-index: 2;
+  left: 80px;
+  top: 15px;
+}
+
+.different-articles-container img:hover {
+  cursor: pointer;
+}
+
+.showcase-mini-story img:hover {
+  cursor: pointer;
+}
+
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `cursor: pointer;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.pointer{
  cursor: pointer;
}
```
**Html file**

```
<div class="pointer">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_